minimint/Cargo.toml

```diff
@@ -33,6 +33,7 @@ hbbft = { git = "https://github.com/fedimint/hbbft" }
 hex = "0.4.2"
 itertools = "0.10.0"
 jsonrpsee = { version = "0.14.0", features = ["ws-server"] }
+mint-client = { path = "../client/client-lib" }
```

Comment:
Ideally we could move the `FederationApi` trait somewhere else … (e.g. core)

Reply:
Will tackle separately.